src/ctia/auth/static.clj

```diff
@@ -50,10 +52,14 @@
                  read-only-capabilities))
   (rate-limit-fn [_ _]))
 
-(defrecord AuthService [auth-config]
+(tk/defservice static-auth-service
   IAuth
-  (identity-for-token [_ token]
-    (let [secret (get-in auth-config [:static :secret])
+  [[:ConfigService get-in-config]]
```

Comment:
There's now a subtle difference between `get-in-config` and `p/get-in-global-properties`. The former is immutable, and the latter is mutable.

I didn't think that was a big enough difference to justify stubbing it in the IAuth services, especially since `get-in-config` is just used during service initialization.